pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[project]
+name = "tails_server"
+version = "1.1.0"
+description = "Tails Server"
+authors = [ { name = "Government of British Columbia", email = "DItrust@gov.bc.ca" } ]
+license = { text = "Apache-2.0"}
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.7"
```

Comment:
Should bump the minimum version of Python up to something that's active.

Reply:
Sure thing

Reply:
I bumped it to the latest version we can use until we fix #70